jobboard/tests.py

```diff
@@ -182,3 +186,57 @@ def test_page_display_to_user_that_is_recruiter(client):
     jobs_to_display = Job.get_jobs_posted_on_or_after_specific_date(date_six_months_ago)
     # test that jobs displayed are only jobs created 6 months ago
     assert response.context['jobs_to_display'] == jobs_to_display
+
+
+@pytest.fixture
+def example_recruiter():
+    example_user = User.objects.create_user("test", "testpassword")
+    example_user.save()
+    recruiter = Recruiter(user=example_user, name="test_recruiter",
+                          company=Company.objects.get(name="example_company_a"), email="test@email",
+                          phone_number="test_phone")
+    recruiter.save()
+    return recruiter
+
+
+@pytest.fixture
+def example_job_data_as_dictionary():
+    example_job = {"title": 'example_job',
+                   "major": student.models.Major.COMPUTER_SCIENCE,
+                   "job_type": 'Full-Time',
+                   "work_from": 'Office-Only',
+                   "description": 'Junior software engineer',
+                   "city": jobboard.models.City.objects.get(name="Tel Aviv-Yafo").id,
+                   "address": 'Hashalom',
+                   "title_keywords": [jobboard.models.JobTitleKeyword.objects.get(keyword="software").id]}
+    return example_job
+
+
+@pytest.mark.django_db
+def test_create_new_job_form_with_valid_data(example_recruiter, example_job_data_as_dictionary,
```

Comment:
Change the parameters in the order that they are used, i.e `client` is used first so it should be the first one

Reply:
will do